frontend/src/components/template/TheHeader.vue

```diff
@@ -17,26 +17,50 @@
 </template>
 
 <script>
+import { useAuthStore } from '../../stores/auth'
+
 export default {
   name: 'TheHeader',
-  data() {
+
+  setup() {
     return {
-      title: config.APP_TITLE || import.meta.env.VITE_APP_TITLE,
+      authStore: useAuthStore(),
     }
   },
   computed: {
     authenticated() {
       return this.$keycloak.authenticated
     },
+
+    title() {
+      let title = ''
+      /**
+    The header is not impacted by which login page was used. It is only impacted by which permission the user has.
+    */
+      if (this.$route.name == 'PBFLogin' || this.authStore.isPBFUser) {
+        title = config.PBF_APP_TITLE || import.meta.env.VITE_PBF_APP_TITLE
+      } else {
+        title = config.APP_TITLE || import.meta.env.VITE_APP_TITLE
```

Comment:
Instead of defining a variable I would just return directly from the if and else.